pymoab/core.pyx

```diff
@@ -24,20 +24,21 @@ cdef class Core(object):
         """MOAB implementation number as a float."""
         return self.inst.impl_version()
 
-    def write_file(self, str fname):
+    def write_file(self, str fname, exceptions = []):
```

Comment:
So, mutable data structures should (almost) never be used as keyword arguments, because the default value carries over between function calls. Thus, this should be a tuple, even if you expect users to pass in a list.  You can make an empty tuple with `()`